src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -161,10 +161,10 @@ public SlaveTemplate(String ami, String zone, SpotConfiguration spotConfig, Stri
             InstanceType type, boolean ebsOptimized, String labelString, Node.Mode mode, String description, String initScript,
             String tmpDir, String userData, String numExecutors, String remoteAdmin, AMITypeData amiType, String jvmopts,
             boolean stopOnTerminate, String subnetId, List<EC2Tag> tags, String idleTerminationMinutes,
-            boolean usePrivateDnsName, String instanceCapStr, String iamInstanceProfile, boolean deleteRootOnTermination,
+            String instanceCapStr, String iamInstanceProfile, boolean deleteRootOnTermination,
             boolean useEphemeralDevices, boolean useDedicatedTenancy, String launchTimeoutStr, boolean associatePublicIp,
-            String customDeviceMapping, boolean connectBySSHProcess, boolean connectUsingPublicIp, boolean monitoring,
-            boolean t2Unlimited) {
+            String customDeviceMapping, boolean connectBySSHProcess, boolean monitoring,
```

Comment:
And here, too. Maybe you can figure out how to support the old booleans by converting them into the enum as appropriate?